cli/src/commands/squash.rs

```diff
@@ -154,6 +154,10 @@ pub(crate) struct SquashArgs {
     #[arg(long, short, conflicts_with = "message_paragraphs")]
     use_destination_message: bool,
 
+    /// Don't open an editor to edit the squashed commit's description
```

Comment:
I'm not sure if it would be better to say why this is useful. Maybe something like

```suggestion
    /// Don't open an editor to edit the squashed commit's description. Instead, all squashed commits descriptions will be combined.
```

Reply:
Oh, I see the linked issue now. I had missed that request and I'm not yet convinced that it's common enough to warrant a flag. Wouldn't you usually want to edit the inputs in some ways, such as by inserting a header at the top or some separator between each?